.github/ISSUE_TEMPLATE/3.documentation.yml

```diff
@@ -0,0 +1,43 @@
+name: 📖 Documentation
+description: Report a problem with the documentation
+labels: ['status: unconfirmed', 'documentation']
+projects: ['folkehelseinstituttet/12']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Thanks for taking the time to report a problem with the documentation!
+
+        > [!TIP]
+        > We accept content written in norwegian if that is your preferred language.
+        > _Vi aksepterer innhold skrevet på norsk hvis det er ditt foretrukne språk._
+
+  - type: input
+    id: link
+    attributes:
+      label: URL to the relevant documentation page
+      placeholder: https://...
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Describe what you were looking for, and what you found problematic or missing in the documentation?
+    validations:
+      required: true
+
+  - type: textarea
+    id: proposed-improvement
+    attributes:
+      label: Proposed improvement
+      description: Your proposed improvement of the spesific part of the documentation that you described above
```

Comment:
Your proposed improvement, if you have any, for the specific part you described above